compiler-test/build.gradle.kts

```diff
@@ -42,6 +42,10 @@ kotlin {
             }
         }
     }
+
+    compilerOptions {
+        freeCompilerArgs.addAll("-P", "plugin:com.kitakkun.backintime.compiler:config=${projectDir.absolutePath}/backintime-default-config.yaml")
```

Comment:
Can this `projectDir.absolutePath` be replaced with `projectDir`...?

Reply:
[Replace projectDir.absolutePath with projectDir](https://github.com/kitakkun/back-in-time-plugin/pull/138/commits/c0a2bff6c21afa22bb69172cc884470dafd01a9f)